cmp/grouping/GroupingChooserModel.ts

```diff
@@ -274,6 +277,11 @@ export class GroupingChooserModel extends HoistModel {
         this.favorites = [...this.favorites, value];
     }
 
+    @action
+    addPendingAsFavorite() {
```

Comment:
With value editor and favorites chooser/add button visible in same panel, I found it more intuitive to have the button add the pending value, which might be a value you just setup but have yet to apply to the view.  This allows you to choose a value you know you will like, add it as a fave, then apply to see results.

The built-in add fave buttons on desktop + mobile now both call this method, and the new `isAddFavoriteEnabled` getter checks if pending val is a fave